### DISCUSSION TITLE
ko/docs/localization/joining-the-localization-team/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#303] [Page comments]
# ko/docs/localization/joining-the-localization-team/

Thanks for being interested on volunteering to translate Scratch Addons to other languages! Scratch Addons is a non-profit open source project where volunteer web developers maintain the extension and create its addons.

https://scratchaddons.com/ko/docs/localization/joining-the-localization-team/

**Comment by rorotube** [comment]
I want to participate in translating scratch addons.
(I am korean)

**Reply by BroJac5246** [reply]
Awesome! Just head [here](https://www.transifex.com/scratch-addons/scratch-addons-extension/) to get started!
